source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
 
   * climate change and extreme weather such as hurricanes
-  * the atmosphere’s chemical makeup and climate
+  * the atmosphere's chemical makeup and climate
 
 * help unlock the climate role played by the oceans that cover nearly
   three-quarters of the globe.
 
-GAEA Quickstart
-===============
-
-This simple overview explains the elements of a basic job in Gaea. It
-includes compiling, running, combining, data transfer, and allocation.
 
-.. Note::
+.. _gaea-node-types:
 
-  Ensure that you have done your first login to Gaea and generated a
-  certificate.
+Node types
+==========
 
-Gaea uses modules software to let users change which software is
-accessible to their environment. There is no module man page. Instead
-use the command:
+Gaea has three node types: :term:`login <login node>` or front-end,
+:term:`compute <compute node>`, :term:`data transfer <data transfer node>`
+nodes (:abbr:`DTN (data transfer node)`).  The three node types are similar in
+terms of hardware, but differ in their intended use.
 
-.. code-block:: shell
-
-  module help
-
-Gaea uses Slurm as its batch scheduler.
-
-Compiling
----------
-
-Gaea offers PrgEnv-intel, Prg-Env-aocc, Prg-Env-nvhpc, and several
-other modules that make it as easy as possible to get your programs
-running. To compile, call either cc or ftn, according to the language
-your code is written in. See **Compilers** for more detail, especially
-for compiling multithreaded applications.
-
-You may compile live in your login shell on a Gaea login node, or in a
-job in the eslogin queue in the es partition of Gaea's batch system.
-To tell a job script to run on the login nodes, specify the following
-in your script:
-
-.. code-block:: shell
-
-  #SBATCH --clusters=es
-  #SBATCH --partition=eslogin_c#
-  #SBATCH --ntasks=1
-
-or, from the sbatch command line:
-
-.. code-block:: shell
-
-  sbatch --clusters=es --partition=eslogin_c# --ntasks=1 /path/to/compile_script
-
-.. note::
++---------+----------------------------------------------------------------+
+| Node    |                                                                |
+| Type    | Description                                                    |
++=========+================================================================+
+| Login   | This is where you are placed when connecting to Gaea. This is  |
+|         | the place to write, edit, and compile your code, manage data   |
+|         | submit jobs, etc. You should not launch parallel or threaded   |
+|         | jobs from a login node. Login nodes are shared resources.      |
++---------+----------------------------------------------------------------+
+| Compute | Most of the nodes on Gaea are compute nodes. These are where   |
+|         | your parallel and threaded jobs execute via the                |
+|         | :command:`srun` command.                                       |
++---------+----------------------------------------------------------------+
```

Comment:
Recommend:

Your parallel and threaded jobs execute in the compute nodes, via the srun command.

Reply:
Fixed in a new push.